integration-tests/scripts/rhtap-e2e-runner.sh

```diff
@@ -97,6 +94,19 @@ configure_gitlab_variables() {
     log "INFO" "GitLab credentials configured successfully (organization: ${GITLAB_ORGANIZATION})"
 }
 
+configure_bitbucket_variables() {
+    log "INFO" "Configuring Bitbucket credentials from cluster secrets"
+
+    if ! secret_exists "rhtap" "rhtap-bitbucket-integration"; then
+        log "WARN" "No Bitbucket integration secret found in the rhtap namespace"
+        return 0
+    fi
+    export BITBUCKET_USERNAME="rhtap-test-admin"
+    export BITBUCKET_WORKSPACE="rhtap-test"
+    export BITBUCKET_PROJECT="RHTAP"
```

Comment:
could you init the above 3 values  through environment variables , you can pass the values in this file https://github.com/redhat-appstudio/rhtap-e2e/blob/main/integration-tests/scripts/rhtap-e2e-runner.sh